processing/proc_MARA.m

```diff
@@ -0,0 +1,299 @@
+function [goodcomp, info] = proc_MARA(cnt_ica,clab,A)
+% Usage:
+%   [goodcomp, info] = proc_MARA(cnt_ica,clab,A)
+%
+%  IN   cnt_ica     - data structure of independent components 
+%                       cnt_ica.x - [nTimePoint x nComponents]
+%                       cnt.fs    - sampling frequency
+%       mnt         - electrode montage structure
+%       A           - mixing matrix
+% 
+%  OUT  goodcomp    - array containing the numbers of the non-artifactual 
+%                     components
+%       info         - struct containing more information about MARA classification 
+%               info.out   : continuous output of the classifier
+%               info.out_p : out transformed into posterior probablity of being an artifact
+%                            P(artifact | out) (assumption normal distribution)
+%               info.fv_te : feature values for each tested component
+%
+% The classifier is based on the following publiction: 
+%
+% Irene Winkler, Stefan Haufe and Michael Tangermann. Automatic Classification of 
+% Artifactual ICA-Components for Artifact Removal in EEG Signals. 
+% Behavioral and Brain Functions, 7:30, 2011.
+%
+% For more information, see: http://www.user.tu-berlin.de/irene.winkler/artifacts/
+%
+% THIS FUNCTION REQUIRES CLASSIFIER .MAT FILES TO BE INCLUDED IN THE PATH. 

```

Comment:
update this description, mentioning that the dependencies can be downloaded with
bbci_import_dependencies('MARA')